### ISSUE TITLE
Hints on command-line switchs please

### ISSUE THREAD

**Issue by lliseil** [brave/browser-laptop#388]
Are there any switches to pass to the browser from command-line?

I tried the common `--help`, `--new-tab $url` (open a new window with a blank page) and some others with no success yet. Would like to include the browser in a real-world application test that relies on opening N pages via a simple script.

Sorry if the answer's present; I just couldn't find it.

**Comment by DamianStanger** [comment]
+1
Id like to hook into my auto hotkey script

**Comment by tukoz** [comment]
Brave v0.8 is just the same total lack in this field.
E.g. no `--help` (nor man page), open in new {tab,window,private-session}, or else.

Any plan for introducing some basic command-line switches?

**Comment by bbondy** [comment]
yes we need this we just haven't added it yet, we should be consistent with other popular browsers at least.

**Comment by tukoz** [comment]
Hey thank for answering! Roadmap?

**Comment by salty-horse** [comment]
It would also be nice to know what `--version` of Brave I have installed without launching it.

**Comment by JamoCA** [comment]
I perform this same function on Windows using Firefox and an "[OpenWith](https://addons.mozilla.org/EN-us/firefox/addon/open-with/)" extension.  This Add-on enables me to instantly launch any other browser using the current URL.  As of 3/8/2017, Brave Browser only opens using the URL IF the browser isn't already open.

- If Brave is already open, nothing happens.
- If Brave is not open, the browser is launched with the URL... and another Brave browser window with the previous session's URL is also opened. ???

I'm currently doing this w/Chrome, IE, Edge, Opera, Safari... Brave is the only browser to misbehave.

**Comment by aliddell** [comment]
Ditto -- have 0.18.16 as my default browser on Ubuntu and xdg-open will exhibit the exact behavior @JamoCA mentioned. I could modify the command line passed to it to maybe fix this -- if I knew what flags I could use.

**Comment by bsclifton** [comment]
@aliddell the issue you mentioned above was originally tracked with https://github.com/brave/browser-laptop/issues/9193

Would you mind leaving some comments there about your setup? (@crysman also didn't have luck with the fix I attempted and is running Xubuntu 16.10).  The fix I attempted was discovered by @DivineOmega and simply updates the `.desktop` file by providing a different `Exec` line (is that file being ignored perhaps?)

Because you can pass Chromium flags to Brave (which are in the `--flagName=value` format), the path argument that would get passed to Brave needs to be escaped.  For example:
`brave -- "pathHere"`

BTW: for folks curious about the various flags, you can check out a full list here:
https://peter.sh/experiments/chromium-command-line-switches/

**Comment by aliddell** [comment]
@bsclifton Sure I will. Thanks for the info.

**Comment by srirambv** [comment]
+1 from fadiphi via https://github.com/brave/brave-browser/issues/391

**Comment by srirambv** [comment]
This should be fixed with brave-core. Command line options are available as out-of-box feature on brave-core

cc: @bsclifton
